config/locales/idv/es.yml

```diff
@@ -31,6 +31,7 @@ es:
       unsupported_jurisdiction: Lo sentimos, no podemos verificar personas de este
         estado.
       unsupported_jurisdiction_sp: Visita %{sp_name} para acceder a tu cuenta.
+      unsupported_otp_delivery_method: NOT TRANSLATED YET
```

Comment:
One option is `Por favor, seleccione un manera de entrega válida.` if we want to start the phrase with "Please".